### ISSUE TITLE
Tuesday

### ISSUE THREAD

**Issue by Codingmaestre2015** [WestMEC-Coding-Session1/Week-25-Bell-Work#2]
Please take a few minutes to review the following website: http://www.entrepreneur.com/article/254103

Read through the article... not word for word, just skim it. 

Choose one of the e-commerce platforms detailed in the article and click on the appropriate link. Explore the website and write a 3-5 sentence summary detailing the finer points of the site. 

Keep in mind, this may be the platform which you might wish to base your e-commerce site off of. Also, someone else might benefit from this site, and your description of it may aid in their development.

**Comment by mason-gavin-wm** [comment]
I read the article Squarespace. Squarespace is a useful ecommerce web creating site were you don't need to know a single line of code to create a beautiful site. Using this you’ll be allowed to upload an unlimited number of items to your online store and have full control of your inventory management. Meaning that it’s easy to offer different variations of the same product type as well as control your inventory stock. This is totally an ecommerce platform for the type of business that you can run even from your bed.

**Comment by burton-jacob-wm** [comment]
I read about Shopify and how they help store owners create functioning websites using their software and more. It is very useful for all businesses because of the app store they control which has 1000 or more free or paid apps to help any business. It talks about how important it is to have a scalable system for business to be able to expand electronically and more.

**Comment by Garcia-Rogelio-WM** [comment]
CommerceHub are services that allows retailers to dramatically increase their inventory.product offering to the consumers that support business.CommerceHub works as a merchandising and fulfillment platform that connects online retailers to suppliers.

**Comment by hogencamp-devon-wm** [comment]
Squarespace is a web hosting service that combines everything you need to get your ecommerce website up and running. They give you templates that you can customize to whatever you want. It also gives other many tools that help you to see traffic and many other analytics. They take care of all the hard work of hosting the site app or whatever.

**Comment by gonzalez-jose-wm** [comment]
Shopify is a well design website that allows companies to use them as an online option to sell whatever product that need. This is very good for these online stores to use this because their setup is easy and well developed to turn any site to a shopping one.

**Comment by clocksin-nick-wm** [comment]
<b>Shopify</b>
The site has pop up social media icons to help interest the user on what sites to shop on. In the online store and how to sell, it describes the online selling process in layman's terms. There is also an easy and basic subscription plan or a complex one that will help one market further.

**Comment by pierce-eric-wm** [comment]
Squarespace is a website builder platform that is user friendly. Squarespace allows you to share your ideas and your creations to the world on your own website for anyone and everyone to see. Squarespace is really good for companies that are starting up and need to get their word out there and it makes  them look more professional when doing so.  I have heard of Squarespace all over youtube with them sponsoring some of the people i watch and also they have reached out to some radios too.

**Comment by shephard-azaniyah-wm** [comment]
The website is easy to access. It also has multiple ways you can sell your merchandise. The list includes Facebook, Pinterest, Twitter, and in person. It is also capable of creating a website for any device and computer platform. They have multiple levels for your online shop starting at 9$ and so forth. This is also easy for your shop to to expand.

**Comment by riggs-jacob-wm** [comment]
I looked at CommerceHub. It allows retailers to increase their inventory and offer support. They work as a merchandising and fulfillment platform. They used cloud based technology. It is great for any start-up that is on pace.

**Comment by steele-ariel-wm** [comment]
I looked at Shopify. Shopify helps people with putting their products online. It makes it very easy to sell your products.

**Comment by mcclure-calvin-wm** [comment]
Squarespace is a website creator. It is convenient for anyone even if they don't have any coding experience. It provides useful templates to keep the business looking professional. There are many tools to view the numbers behind the site as well. You have full control, but in a very simplified fashion.

**Comment by diaz-andy-wm** [comment]
The Magento website immediately boasts being first as an internet retailer in several awards. The website functionality is nicely put together and the interactivity within is also impressive. It has a nice set of clients which are obviously very successful today. The site overall looks trustworthy and attractive

**Comment by le-ben-wm** [comment]
I explored Symphony Commerce. It helps retailers transfer from a physical store to an online store. I found that it prioritizes creating a mobile interface along with a normal web interface, which enables better accessibility no matter where the customer is.

**Comment by marquez-mollee-wm** [comment]
I chose Magento and from what I understood it is a way for businesses to manage the online aspects of their company. They are an open source company so a user can pick and choose and change what the program can do. They have an extension that can track what customers are doing on the site and tailor the site for that persons needs.

**Comment by deherrera-marques-wm** [comment]
I read about Shopify and they help store owners create websites. You can sell things on social media, and makes selling items a lot easier. It reminds me of amazon, but it was an interesting site to look at it had a great design.

**Comment by schweikert-zachary-wm** [comment]
Shopify has many decorative aspects that attract the consumer. They allow people to create their own business with the ability to use debit and credit cards. Shopify already has a vast amount of customers so advertisement will spread by word of mouth pretty easily.

**Comment by brey-matthew-wm** [comment]
Symphony Commerce is basically catered towards small businesses. It is so they can pay it professionals as they go to manage their website so they can focus more on what they want. It makes it so they don't have to manage code themselves, and it makes things simple.

**Comment by capp-brendan-wm** [comment]
The e-commerce platform I chose was Shopify.

This platform lets people sell goods through them, sort of like a bridge, they even have connections with 3 major social media platforms being Facebook, Twitter, and Pinterest. They give multiple different ways to sell the products and they even help with the selling of the products. They also give customization of when someone lists their product to sell, they can customize a theme around the product.

**Comment by osborn-daucen-wm** [comment]
### Shopify: 
Shopify is an ecommerce site that provides a platform for industries. Over 100,000 companies have reported using Shopify and boast about the multiple choices that they have. With it you are able to manage your products with over 1,000 apps.

**Comment by Keyan-Ngai-wm** [comment]
Squarespace is a site that creates layouts for small businesses and people trying to sell services. The great thing about squarespace is that it creates really great layouts/platforms. Squarespace is also great cause it allows the user to upload and edit their inventory as much as they want. Which is good cause that would give the user full control over their inventory.

**Comment by thorpe-sean-wm** [comment]
Shopify is the ecommerce website that I read about. Shopify is a ecommerce platform that is focused on "helping you set up an online store so that you can sell online". The website's creativity and interactivity make it unique from the others all while following the color scheme of green, white and black. Right off the start users can go into depth about how to sell off their website, subscriptions, Shopify blogs and a drop down menu for "More".
